packages/linear-genome-view/src/BasicTrack/components/ServerSideRenderedBlockContent.tsx

```diff
@@ -81,15 +79,10 @@ BlockMessage.propTypes = {
   messageText: PropTypes.string.isRequired,
 }
 
-function BlockError({ error, reload }: { error: Error; reload: () => void }) {
+function BlockError({ error }: { error: Error }) {
   const classes = useStyles()
   return (
     <div className={classes.blockError}>
-      {reload ? (
-        <Button onClick={reload} size="small" startIcon={<RefreshIcon />}>
-          Reload
-        </Button>
-      ) : null}
```

Comment:
is this deletion of reload supposed to be here?

Reply:
basically yes, it wasn't part of my explanation of the PR but I wanted to remove block level reloading, instead relying on Peter's soon to be made reload PR